tests/data/tensorflow_mnist/mnist_v2.py

```diff
@@ -0,0 +1,204 @@
+# Copyright 2018-2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+import argparse
```

Comment:
This is a new training script that is functionally the same as the `mnist_v1.py` script, except that it does not use `tensorflow.v1.compat` APIs. The v1 compat APIs have been removed from TF>=2.6.0, which necessitates the use of a different training script.